packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
+-   We updated the typings generator for icon properties to support icon collections, introduced in Mendix 9.24.
+
+### Breaking changes
+
+-   Icon properties now use `undefined` instead of `null` when no icon is selected.
```

Comment:
We could add that this was done for compatibility with the `Icon` component in the pluggable widgets API.